detox/android/detox/src/main/java/com/wix/detox/DetoxActionsFacade.kt

```diff
@@ -0,0 +1,11 @@
+package com.wix.detox
+
+import android.content.Context
+import android.support.test.espresso.Espresso
+import com.wix.detox.espresso.UiAutomatorHelper
+
+class ActionsFacade {
```

Comment:
What did we do wrong to have a `Facade` as a class name? 😣

Reply:
Lol, I don't care much about names. It's something that simplifies and aggregates other things have spread around in other places (mainly helpers), instead of providing its own direct implementation (which is a facade basically 🤔). In our case, it also makes actions unit-testable. wdyt?